HabitTracker.AshtonLeeSeloka/DBItem/DBItems.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace DBItem
+{
+	public class DBItems
```

Comment:
🟠 **Naming Conventions**

💡 What is a `DBItems`? Would this not be better described as a `Habit`, `HabitRecord`, `HabitItem`?

Reply:
HI Chris.

Thanks for the feedback , I have updated this class name to Habit.